TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift

```diff
@@ -25,16 +25,33 @@ class TradeItTradingTicketViewController: TradeItViewController, TradeItSymbolSe
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        if self.order.linkedBrokerAccount == nil {
+        guard let linkedBrokerAccount = self.order.linkedBrokerAccount else {
             assertionFailure("TradeItIosTicketSDK ERROR: TradeItTradingTicketViewController loaded without setting linkedBrokerAccount on order.")
+            return
         }
         
         orderActionSelected(orderAction: TradeItOrderActionPresenter.labelFor(order.action))
         orderTypeSelected(orderType: TradeItOrderPriceTypePresenter.labelFor(order.type))
         orderExpirationSelected(orderExpiration: TradeItOrderExpirationPresenter.labelFor(order.expiration))
-        
-        updateSymbolView()
-        updateTradingBrokerAccountView()
+
+        linkedBrokerAccount.linkedBroker.authenticateIfNeeded(onSuccess: {
+            linkedBrokerAccount.getAccountOverview(onSuccess: {
+                self.updateSymbolView()
+                self.updateTradingBrokerAccountView()
+            }, onFailure: { errorResult in
+                print(errorResult)
+            })
+        }, onSecurityQuestion: { securityQuestion, answerSecurityQuestion, cancelQuestion in
+            self.alertManager.promptUserToAnswerSecurityQuestion(
+                securityQuestion,
+                onViewController: self,
+                onAnswerSecurityQuestion: answerSecurityQuestion,
+                onCancelSecurityQuestion: cancelQuestion
+            )
+        }, onFailure: { errorResult in
+            print(errorResult)
```

Comment:
Something needs to happen here:
- take user to account selection screen?
- show alert with error?
- relink if necessary?
- retry?  This is "if needed" so maybe authentication session just expired...